dali/pipeline/operators/resize/resize_test.cc

```diff
@@ -16,35 +16,114 @@
 
 namespace dali {
 
+// Values of eps for different tests and different methods for calculation of average deviation
+// When flag t_checkElements is NOT set, the average deviation is calculated COMBINED for ALL
+//      images of the batch - the eps from first column will be used.
+// When flag t_checkElements is set, the average deviation is calculated SEPARATELY for EACH image
+//      of the batch - the eps from second column will be used.
+
+static double testEps[] = {     //     TEST:
+                    0.7, 1.8,   // ResizeShorter_LINEAR
```

Comment:
As I understand right, the first value is eps for each channel separately, the second one is combined. So it is not supposed to be b = 3 * a, or a = b / 3?

Reply:
No, it is not that. When `t_checkElements` flag is set, than all images are tested separately against the eps. Otherwise, they are tested all together ("combined"). It is also possible to test all channels separately. To do that, you need to use `t_checkColorComp` flag.

Reply:
Understood